src/ReactiveUI/ReactiveUI.csproj

```diff
@@ -57,7 +57,7 @@
   
   <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.0' ">
     <Compile Include="Platforms\netcoreapp1.0\**\*.cs" />
-    <PackageReference Include="System.Runtime.Serialization.Primitives" Version="4.1.1" />
+    <PackageReference Include="System.Runtime.Serialization.Primitives" Version="4.3.0" />
```

Comment:
This causes package downgrade warnings. Please don't change this for netcoreapp1.x.

Reply:
It being 4.1.1 also causes issues with splat3 I think (https://ci.appveyor.com/project/dotnetfoundation/reactiveui/build/1.0.442#L37419)

Reply:
That's because splat is broken: 

https://github.com/reactiveui/splat/blob/3.0.0/src/Splat/Splat.csproj#L19

Should not be referencing 4.3 there either.

Best solution overall is to remove netcoreapp1.0 and netstandard1.x. min version should be net461 and netstandard2.0.